tools/networkmanager/inject_txs.go

```diff
@@ -183,10 +185,10 @@ func checkDepositsSuccessful(txInjector *simulation.TransactionInjector, l1Clien
 	}
 }
 
-func checkL2TxsSuccessful(l2Client rpcclientlib.Client, txInjector *simulation.TransactionInjector) {
+func checkL2TxsSuccessful(rpcHandles *network.RPCHandles, txInjector *simulation.TransactionInjector) {
```

Comment:
@BedrockSquirrel Can you add a todo to investigate why we do some validation here, then repeat it in `validate_chain.go`?

Reply:
This code is in the network manager tool (it's not the sim transaction injector code).

It re-uses a bit of the validate_chain logic from the sims to check the transactions behaved as successful but I don't think `validate_chain.go` gets called in full afterwards.

Reply:
Ah thanks. Got confused between the two inject_txs.go's.